python-threatexchange/threatexchange/signal_type/raw_text.py

```diff
@@ -41,15 +41,12 @@ def get_content_types(cls) -> t.List[t.Type[ContentType]]:
 
     @classmethod
     def matches_str(
-        cls, signal: str, haystack: str, distance_threshold: t.Optional[int] = None
+        cls, signal: str, haystack: str, pct_diff_threshold: int = 5
```

Comment:
nit: create a constant for this default threshold 5

Reply:
I prefer not to create a constant if it's only used in one place, otherwise you are just adding indirections in the code!